contracts/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "kava-contracts",
+  "version": "0.0.1",
+  "author": "Kava Labs",
+  "private": true,
+  "description": "Solidity contracts for Kava Blockchain",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "build": "npm run clean && npm run compile && npm run ethermint-json",
+    "clean": "hardhat clean",
+    "compile": "hardhat compile",
+    "coverage": "hardhat coverage",
+    "ethermint-json": "jq '{ abi: .abi | tostring, bin: .bytecode | ltrimstr(\"0x\")}' artifacts/contracts/ERC20KavaWrappedNativeCoin.sol/ERC20KavaWrappedNativeCoin.json > ../x/evmutil/types/ethermint_json/ERC20KavaWrappedNativeCoin.json",
+    "gen-ts-types": "hardhat typechain",
+    "lint": "eslint '**/*.{js,ts}'",
+    "lint-fix": "eslint '**/*.{js,ts}' --fix",
+    "prettier": "prettier '**/*.{json,sol,md}' --check",
+    "prettier-fix": "prettier '**/*.{json,sol,md}' --write",
+    "solhint": "solhint 'contracts/**/*.sol' --max-warnings 0",
+    "solhint-fix": "solhint 'contracts/**/*.sol' --fix",
+    "test": "hardhat test"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-toolbox": "^2.0.2",
+    "@openzeppelin/contracts": "4.8.3",
```

Comment:
using a fixed verison of the open zeppelin contracts package (note the lack of `^` 🥕 )